### DISCUSSION TITLE
[Help Wanted] useFieldArray - have warning that re-renders my React-native

### DISCUSSION THREAD

**Discussion by manongguard** [react-hook-form/react-hook-form#3860] [Q&A]
Hi I have been reading on this https://react-hook-form.com/api/#useFieldArray

But i always have this warning that re-renders my application again

`Field is missing 'name' attribute [object Object] https://react-hook-form.com/api#useForm`

**Comment by bluebill1049** [comment]
are you using Controller? did you give a name attribute?

**Reply by manongguard** [reply]
The first one I made was a Controller with a name on it but I am having the other errors for that.

I have used `useFieldArray` in a `Controller` component but has a warning that it wants something to do about a `ref`

so I converted it into a another normal `<TextInput />` component

`<TextInput
            name={'someA.someB[${index}].someC'}
            style={styles.input}
            autoCapitalize={'words'}
            ref={register()}
            defaultValue={field.someC}
          />`

But I am still having warning about Field not having a name. any tips?

**Reply by bluebill1049** [reply]
can you reproduct that in a codesandbox?
